smartsim/_core/launcher/launcher.py

```diff
@@ -140,7 +143,13 @@ def get_step_update(
         s_names, step_ids = self.step_mapping.get_ids(step_names, managed=True)
         if len(step_ids) > 0:
             _step_ids = [str(sid) for sid in step_ids]
-            s_statuses = self._get_managed_step_update(_step_ids)
+            s_statuses: t.List[StepInfo] = []
+
+            try:
+                s_statuses = self._get_managed_step_update(_step_ids)
+            except Exception:
+                logger.warning(f"Step update retrieval failed for: {_step_ids}")
```

Comment:
Why are we silencing this exception? I'm not sure I want to commit to making this API break. Is it strictly necessary for the authenticator changes?

Reply:
I found that a bad step ID could cause the telemetry monitor / JobManager to go down in flames... But on reflection, that's usually due to not cleaning up old manifests.

Removed.